src/legacy/core_plugins/kibana/server/tutorials/nats_metrics/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n }  from '@kbn/i18n';
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/metricbeat_instructions';
+
+export function natsMetricsSpecProvider(server, context) {
+  const moduleName = 'nats';
+  return {
+    id: 'natsMetrics',
+    name: i18n.translate('kbn.server.tutorials.natsMetrics.nameTitle', {
+      defaultMessage: 'Nats metrics',
+    }),
+    category: TUTORIAL_CATEGORY.METRICS,
+    shortDescription: i18n.translate('kbn.server.tutorials.natsMetrics.shortDescription', {
+      defaultMessage: 'Fetch monitoring metrics from the Nats server.',
+    }),
+    longDescription: i18n.translate('kbn.server.tutorials.natsMetrics.longDescription', {
+      defaultMessage: 'The `nats` Metricbeat module fetches monitoring metrics from Nats. \
+[Learn more]({learnMoreLink}).',
+      values: {
+        learnMoreLink: '{config.docs.beats.metricbeat}/metricbeat-module-nats.html',
+      },
+    }),
+    euiIconType: 'logoNats',
+    artifacts: {
+      dashboards: [
+        {
+          id: 'Metricbeat-Nats-Dashboard',
```

Comment:
I think this needs adjustement as the id changed slightly on master: https://github.com/elastic/beats/blob/master/metricbeat/module/nats/_meta/kibana/7/dashboard/Metricbeat-nats-overview.json#L1682

Reply:
Hi I would like to mention something here. I did not include the -ecs on purpose because I found out this one:

https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/server/tutorials/apache_metrics/index.js#L46
https://github.com/elastic/beats/blob/master/metricbeat/module/apache/_meta/kibana/7/dashboard/Metricbeat-apache-overview.json#L775

At every dashboard id of Kibana tutorial -ecs is missing

Reply:
You are correct and thanks to your PR I realised that: https://github.com/elastic/kibana/issues/30805

Reply:
Cool :) Let's wait for Jenkins to become green then

Reply:
yeah, Jenkins ...